README.md

```diff
@@ -24,20 +21,36 @@ DB_USER = "homer_simpson"
 DB_PASSWORD = "Close, but you're way off."
 ```
 
+If using _.env.yml_ file format
+```
+DB_NAME: "sector_seven"
+DB_USER: "homer_simpson"
+DB_PASSWORD: "Close, but you're way off."
+```
+
 In the project settings script, include this snippet:
 ```python
 from django_envie.workroom import convertfiletovars
 
-
 convertfiletovars()
 ```
 
-Use
-----
+## Accessing your environment variables
 Accessing environment variables anywhere can be done by using
 ```python
 os.getenv('DB_NAME')
 ```
+## Setting up your environment variable for Continuous Integration
+### CircleCI
+```
+KEY = SECRET_KEY
+VALUE = my secret key
+```
+### TravisCI
+```
+KEY = SECRET_KEY
```

Comment:
Nice. It might be more understood should you have it written that setting Travis CI environment variables of KEY-VALUE pairs, VALUES must be quoted as shown.